website/docs/sysincludedirs.md

```diff
@@ -4,9 +4,12 @@ Specifies the system include file search paths.
 sysincludedirs { "paths" }
 ```
 
-For Visual Studio, these paths are placed in the "VC++ Directories" properties panel. For GCC and Clang, they are preceded with the `-isystem` flag, rather than `-I`. For toolsets which do not support the concept of system include directories, they are treated as a normal include directory.
+For Visual Studio, these paths are placed in the "VC++ Directories" properties panel under "External Include Directories". Note that unlike gcc and clang this does not control warning levels. See externalwarnings to control the warning level for sysincludes in visual studio.
```

Comment:
```suggestion
For Visual Studio, these paths are placed in the "VC++ Directories" properties panel under "External Include Directories". Note that unlike gcc and clang this does not control warning levels. See [externalwarnings](externalwarnings.md) to control the warning level for sysincludes in Visual Studio.
```

Reply:
In addition to @KyrietS changes, it would be useful to know what "unlike gcc and clang this does not control warning levels" means. What do GCC and Clang do differently? I don't know what GCC & Clang do TBH but assuming they suppress warnings in system headers I might put the important bit first and say "To control warning levels for system includes on Visual Studio, see [externalwarnings]; other toolsets will suppress warnings and errors in system includes."